partcad/src/partcad/wrappers/custom_cqgi.py

```
@@ -488,16 +488,14 @@ def __init__(self, cq_model):
+        # actually handles numbers and strings.
         try:
-            if type(value_node) == ast.Num:
-                self.cqModel.add_script_parameter(
-                    InputParameter.create(value_node, var_name, NumberParameterType, value_node.n)
-                )
-            elif type(value_node) == ast.Str:
-                self.cqModel.add_script_parameter(
-                    InputParameter.create(value_node, var_name, StringParameterType, value_node.s)
-                )
-            elif type(value_node) == ast.Name:
+            if type(value_node) == ast.Name:
```

Ruff (0.15.21)

[error] Use `is` and `is not` for type comparisons, or `isinstance()` for isinstance checks

(E721)